tests/expectedoutputs/TestWMS/test_WMSGetCapabilities_KMDS_PointNetCDF_pointstylepoint.xml

```diff
@@ -135,51 +185,51 @@
   <eastBoundLongitude>7.149322</eastBoundLongitude>
   <southBoundLatitude>12.130000</southBoundLatitude>
   <northBoundLatitude>55.399166</northBoundLatitude>
-</EX_GeographicBoundingBox><BoundingBox CRS="EPSG:3411" minx="-3942117.046146" miny="-9975996.132113" maxx="7877215.337643" maxy="-2369697.552660" />
+</EX_GeographicBoundingBox><BoundingBox CRS="CRS:84" minx="12.130000" miny="-68.275833" maxx="55.399166" maxy="7.149322" />
+<BoundingBox CRS="EPSG:25831" minx="-9906758.676129" miny="1341055.638469" maxx="951836.432640" maxy="8605856.308089" />
+<BoundingBox CRS="EPSG:25832" minx="-11426227.343341" miny="1341612.511753" maxx="382800.244818" maxy="9034799.679759" />
+<BoundingBox CRS="EPSG:28992" minx="-8805606.109966" miny="-4164572.069133" maxx="372143.463211" maxy="3233625.747402" />
+<BoundingBox CRS="EPSG:3067" minx="-13588402.263697" miny="1423498.626368" maxx="-747816.881762" maxy="12453122.100930" />
+<BoundingBox CRS="EPSG:32661" minx="-7498001.230434" miny="-8223848.076474" maxx="3272454.133168" maxy="535002.459267" />
+<BoundingBox CRS="EPSG:3411" minx="-3942117.046146" miny="-9975996.132113" maxx="7877215.337643" maxy="-2369697.552660" />
 <BoundingBox CRS="EPSG:3412" minx="-36574088.915720" miny="5641285.215970" maxx="4899857.294031" maxy="39369117.726749" />
 <BoundingBox CRS="EPSG:3575" minx="-7844965.759495" miny="-8002206.437441" maxx="-189084.750364" maxy="-772564.105215" />
 <BoundingBox CRS="EPSG:3857" minx="-7600430.977505" miny="1360506.839359" maxx="795858.888223" maxy="7439724.721958" />
+<BoundingBox CRS="EPSG:40000" minx="-9555333.229813" miny="-10285561.445145" maxx="1280134.942825" maxy="-1473840.584239" />
 <BoundingBox CRS="EPSG:4258" minx="12.130000" miny="-68.275833" maxx="55.399166" maxy="7.149322" />
 <BoundingBox CRS="EPSG:4326" minx="12.130000" miny="-68.275833" maxx="55.399166" maxy="7.149322" />
```

Comment:
The CRS's get sorted alphabetically, would it be good idea to sort numerically? That way `CRS:3067` would come before `CRS:25831`.

Not sure if it's an advantage for a client who has to parse this, current solution works fine as well.

Reply:
I would like to leave as is, I think it does not cause any issues and it is consistent like this.

Reply:
Ok, super! Note that this MR will help fix: https://github.com/KNMI/adaguc-server/issues/350